packages/zoe/test/unitTests/contracts/test-scaledPriceAuthority.js

```diff
@@ -63,11 +63,11 @@ test.before(
 
     // Pick some weird decimal places.
     const dp = decimalPlaces => harden({ decimalPlaces });
-    const { brand: atomBrand } = makeIssuerKit('$ATOM', undefined, dp(5));
-    const { brand: usdBrand } = makeIssuerKit('$USD', undefined, dp(4));
+    const { brand: atomBrand } = makeIssuerKit('$ATOM', 'nat', dp(5));
```

Comment:
I think there's an `AssetKind` enum around somewhere, can we use that?

Reply:
We could. But this is a regression test so if perchance the enum value changed the test should fail.